### ISSUE TITLE
Improve toggle design

### ISSUE THREAD

**Issue by sradevski** [balena-io-modules/rendition#1182]
The current toggle doesn't clearly convey its state, and we already have updated designs to improve the situation.

**Comment by codewithcheese** [comment]
That's correct. The user is required to know that switch to the right is enabled which is commonly held knowledge. 

![image](https://user-images.githubusercontent.com/989237/99801460-2ddef600-2b8a-11eb-9163-3e95f2369488.png)

Usually a toggle will have a colored background to indicate it is "on" vs uncolored which is "off".

![image](https://user-images.githubusercontent.com/989237/99801399-13a51800-2b8a-11eb-937c-9dad29618845.png)
